### PR TITLE
Work/03 

### PR DESCRIPTION
**Pull request by Sahilchouhan20**

### CODE REVIEW

File: app/assets/stylesheets/home_index.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
+    @import url(https://fonts.googleapis.com/css?family=Dancing+Script);
+      * {
+          margin: 0;
+      }
+
+      body {
+          background-color: #e8f5ff;
+          font-family: italic;
+
```
Comment (rishabhharod) [inline app/assets/stylesheets/home_index.css:9]: avoid the extra space

File: app/assets/stylesheets/home_search.css [hunk 2]
```diff
@@ -0,0 +1,60 @@
+@import url(https://fonts.googleapis.com/css?family=Dancing+Script);
+
+      * {
+          margin: 0;
+      }
+
+      body {
+          background-color: #e8f5ff;
+          font-family: italic;
+
```
Comment (rishabhharod) [inline app/assets/stylesheets/home_search.css:10]: remove the extra space

File: app/assets/stylesheets/home_search.css [hunk 3]
```diff
@@ -0,0 +1,60 @@
+@import url(https://fonts.googleapis.com/css?family=Dancing+Script);
+
```
Comment (rishabhharod) [inline app/assets/stylesheets/home_search.css:2]: import this url in application.css

File: app/controllers/chats_controller.rb [hunk 4]
```diff
@@ -0,0 +1,65 @@
+class ChatsController < ApplicationController
+  def start_chat
+    @user = current_user
+    @other_user = User.find(params[:other_user_id])
+    @chat = find_or_create_chat
+    redirect_to @chat
+  end
+
+  def show
+    @user = current_user
```
Comment (rishabhharod) [inline app/controllers/chats_controller.rb:10]: you should define the current user in application controller

File: app/controllers/chats_controller.rb [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+    @message = Message.new
+    @message.user = current_user
+    #user 1 is the current user, now we need to find the other user
+    @other_user = User.find(@other_user_id) if @other_user_id #the if here makes sure the app doesnt break if the other user doesnt exist
+  end
+private
+  def find_or_create_chat
+    # Get the current user's ID
+    @user_id = current_user.id.to_i
+
```
Comment (rishabhharod) [inline app/controllers/chats_controller.rb:24]: could you explain why you parse the id

File: app/controllers/stories_controller.rb [hunk 6]
```diff
@@ -3,12 +3,15 @@ class StoriesController < ApplicationController
   before_action :instance_story
 
   def index
+    debugger
```
Comment (rishabhharod) [inline app/controllers/stories_controller.rb:6]: remove the debugger

File: app/views/chats/_chat_bar.html.erb [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+.mask-custom {
+background: rgba(24, 24, 16, .2);
+border-radius: 2em;
+backdrop-filter: blur(15px);
+border: 2px solid rgba(255, 255, 255, 0.05);
+background-clip: padding-box;
+box-shadow: 10px 10px 10px rgba(46, 54, 68, 0.03);
+}
+</style>
+
```
Comment (rishabhharod) [inline app/views/chats/_chat_bar.html.erb:22]: add css in css file

File: app/assets/stylesheets/home_index.css (hunk 1 above)
Comment (Sahilchouhan20) [reply app/assets/stylesheets/home_index.css:9]: okk i will change it

File: app/assets/stylesheets/home_search.css (hunk 3 above)
Comment (Sahilchouhan20) [reply app/assets/stylesheets/home_search.css:2]: okk

File: app/assets/stylesheets/home_search.css (hunk 2 above)
Comment (Sahilchouhan20) [reply app/assets/stylesheets/home_search.css:10]: okk i will change this

**Pull request merged**